utils/helper.js

```diff
@@ -172,4 +173,8 @@ export function getParentsListFromPath(path) {
     })
 }
 
-export default normalizedSidebar
+module.exports = {
+  sidebar: normalizedSidebar,
+  getItemByPath,
+  getParentsListFromPath
+}
```

Comment:
Is this what you meant by changing file to CommonJS? Just checking 🙂

Reply:
Yep. `import` changed to `require` and `export` to `module.exports`. Also replaced `default` export with the named one.